### PR TITLE
Added info about normalization costs to docs

### PR DESCRIPTION
**Pull request by sophia-wiley**
Added information about normalization costs. Also removed information about dbt cloud transformations not being available yet. 

Related to this [PRD](https://docs.google.com/document/d/1A7WOxRAGhCbQcPM6MIcwE8nEKlztLe1Mm-bbq3bizps/edit#)

### CODE REVIEW

File: docs/cloud/core-concepts.md [hunk 1]
```diff
@@ -146,8 +146,13 @@ After a sync is complete, Airbyte normalizes the data. When setting up a connect
 
 * Raw data (no normalization): Airbyte places the JSON blob version of your data in a table called `_airbyte_raw_<stream name>`
 * Basic Normalization: Airbyte converts the raw JSON blob version of your data to the format of your destination. *Note: Not all destinations support normalization.*
+* [dbt Cloud integration](https://docs.airbyte.com/cloud/dbt-cloud-integration): Airbyte integrates dbt Cloud, allowing you to use dbt Cloud for transforming and cleaning your data during the normalization process.
```
Comment (sophia-wiley) [inline docs/cloud/core-concepts.md:149]: @mfsiega-airbyte Is this accurate?

File: docs/cloud/core-concepts.md [hunk 2]
```diff
@@ -146,8 +146,13 @@ After a sync is complete, Airbyte normalizes the data. When setting up a connect
...
 
-*Note: Custom normalization through dbt is not yet available for Airbyte Cloud.*
+:::note
+
+Normalizing data may cause an increase in your destination's compute cost. This cost will vary depending on the amount of data that is normalized and is not related to Airbyte credit usage.
```
Comment (sophia-wiley) [inline docs/cloud/core-concepts.md:153]: @andyjih is there any other info you think users should know about normalization costs?

Comment (andyjih) [reply docs/cloud/core-concepts.md:153]: I think this looks good to me!

**Review by andyjih — APPROVED** [review]

File: docs/cloud/core-concepts.md (hunk 1 above)
Comment (mfsiega-airbyte) [reply docs/cloud/core-concepts.md:149]: Looks good to me!

**Review by mfsiega-airbyte — APPROVED** [review]

**Review by Amruta-Ranade — APPROVED** [review]

**Pull request merged**